addons/mail/__manifest__.py

```diff
@@ -149,6 +149,9 @@
             'mail/static/src/components/notification_list/notification_list_item.scss',
             'mail/static/src/component_hooks/*.js',
             'mail/static/src/components/*/*',
+
+            # Don't include dark mode files in portal as portal user does not need to have scss for dark theme
+            ('remove', 'mail/static/src/components/*/*.dark.scss'),
```

Comment:
Comment is useless, we know that `*.dark.scss` files are specific to `dark_mode_assets_backend` hence their removal in base bundle.

Also could just target `mail/static/src/**/*.dark.scss` so that it removes all `*.dark.scss` in `mail/`, which is handy for forward-ports

Reply:
@alexkuhn 
Thanks for the review. 
I have updated the code.